joomla/joomla-cms
# remove unused mysettings layout

Pull Request for Issue #15064 .

### Summary of Changes
As discussed in 15064 this layout is no longer being used and this PR removes it


### Testing Instructions
Delete the layout and make sure that the My Settings button in com_messages still works
